packages/hawtio/src/plugins/springboot/Health.tsx

```diff
@@ -32,21 +32,41 @@ const ComponentDetails: React.FunctionComponent<{ componentDetails: HealthCompon
           )
         })}
       </Tbody>
-    </TableComposable>
+    </Table>
   )
 }
 const HealthStatusIcon: React.FunctionComponent<{ status: string }> = ({ status }) => {
   switch (status) {
     case 'UP':
-      return <CheckCircleIcon color={'green'} />
+      return (
+        <Icon color={'green'}>
```

Comment:
If your comment in another place is right, shouldn't then this be changed to use the css style directly?